src/embed.ts

```diff
@@ -257,17 +257,20 @@ export default async function embed(
     view.tooltip(handler);
   }
 
-  const hover = opt.hover;
+  let hover = opt.hover;
 
   // do not automatically enable hover for Vega-Lite.
```

Comment:
The code now does "enable hover for Vega if not specified"

Reply:
@domoritz Is that incorrect...?

Reply:
Well, the code doesn't use the negation so the comment shouldn't either.

Reply:
I see. I misunderstood that you talking about the code and not the comment.